### PR TITLE
doc(Pages): Update remaining pages to remove reference to Portal

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [RUNTIME-525](https://bonitasoft.atlassian.net/browse/RUNTIME-525)

### CODE REVIEW

File: modules/ROOT/pages/logging.adoc [hunk 1]
```diff
@@ -7,13 +7,12 @@
 
 The components of Bonita have their own ways to produce logs, which are described below.
```
Comment (vbotnari12) [inline modules/ROOT/pages/logging.adoc]: The components of Bonita => Bonita components

Comment (vbotnari12) [reply modules/ROOT/pages/logging.adoc]: which are described below. => as described below.

File: modules/ROOT/pages/logging.adoc [hunk 2]
```diff
@@ -7,13 +7,12 @@
...
 
-=== Bonita Engine
-
-All the Engine services uses SLF4J in order to write log messages.
+=== Bonita Runtime
 
-=== Bonita Portal
+The Bonita Runtime is made out of two pars :
```
Comment (vbotnari12) [inline modules/ROOT/pages/logging.adoc]: made out = make out = to deal with a situation, usually in a successful way. 

The Bonita Runtime is made out of two pars : = > The Bonita Runtime has two components:

File: modules/ROOT/pages/logging.adoc [hunk 3]
```diff
@@ -86,7 +85,7 @@ protected void executeBusinessLogic() throws ConnectorException {
 In Bonita Studio you have access to two log files:
```
Comment (vbotnari12) [inline modules/ROOT/pages/logging.adoc]: to two log files: => to the following log files:

File: modules/ROOT/pages/multi-language-pages.adoc [hunk 4]
```diff
@@ -41,15 +41,15 @@ To use a localization asset, replace the default `localization.json` file with a
   ...
 ----
 
-The code that specifies a language in `localization.json` must be the same as the code in the `.po` files used to xref:languages.adoc[add the language to the Portal].
+The code that specifies a language in `localization.json` must be the same as the code in the `.po` files used to xref:languages.adoc#_add_a_new_language[add a new language].
 
 By default, US english strings are used as the keys for translation. But you could use any code or language for the keys, and create an en-US section of translations to consider english as any other translated language. That option allows to freely change english strings with no duplication on every other language section.
```
Comment (vbotnari12) [inline modules/ROOT/pages/multi-language-pages.adoc]: By default, US english strings are used as the keys for translation. But you could use any code or language for the keys, and create an en-US section of translations to consider english as any other translated language. That option allows to freely change english strings with no duplication on every other language section. => By default, translation keys are in en-US english texts. If you want to have custom texts, you can create a new translation section for en-US and handle it as a new translation. This option allows to freely change texts by avoiding duplicates on other language sections.

File: modules/ROOT/pages/profile-creation.adoc [hunk 5]
```diff
@@ -82,10 +82,7 @@ You can duplicate a profile name in different files, but remember that you can o
 In the mapping section you can map Users / Groups / Roles / Memberships to your profile. You have to map your profile with elements of the active organization. +
```
Comment (vbotnari12) [inline modules/ROOT/pages/profile-creation.adoc]: of the active organization. => of an active organization.

File: modules/ROOT/pages/profile-creation.adoc [hunk 6]
```diff
@@ -82,10 +82,7 @@ You can duplicate a profile name in different files, but remember that you can o
...
 ⚠  Nodes in the group tree are *independent* ⚠  It means that mapping `/acme` to your profile won't map `/acme/hr` to it. Each node has to be mapped individually to be taken into account. However, you can easily add a whole part of your organization, using the _add subtree_ button:  select a node in the tree, and then click on _add subtree_.
 
-The portal menu entries can be edited for custom profiles. +
-Pages proposed are those provided by default in the Bonita Runtime and those created with the UI Designer. You can reference a custom page only if it is imported in the Bonita Runtime, and you will have to write its exact identifier.
-
-The Applications section provides a quick access to the applications mapped to this profile. +
+The Applications section provides quick access to the applications mapped to this profile. +
 To map a new application to this profile, you have to open the application and then choose this profile in the profile field of the xref:applicationCreation.adoc[application descriptor editor]
```
Comment (vbotnari12) [inline modules/ROOT/pages/profile-creation.adoc]: profile, too much profile. 

To map a new application to this profile, you have to open the application and then choose this profile in the profile field of the xref:applicationCreation.adoc[application descriptor editor] => 
For adding an application to a profile, open the application and assign the required profile by using xref:applicationCreation.adoc[application descriptor editor].

File: modules/ROOT/pages/profiles-applications-overview.adoc [hunk 7]
```diff
@@ -35,22 +35,12 @@ By default, it gives access to the Bonita User Application.
 
 By default, its gives access to Bonita Administrator Application. It includes:
```
Comment (vbotnari12) [inline modules/ROOT/pages/profiles-applications-overview.adoc]: By default, its gives access to Bonita Administrator Application. It includes:  => By default, it gives access to Bonita Administrator Application, which includes:

File: modules/ROOT/pages/run-a-process-from-bonita-bpm-studio-for-testing.adoc [hunk 8]
```diff
@@ -14,16 +14,4 @@ To run a process, you can :
...
-* Standalone mode uses templates to contain the form application, These templates are editable in Studio. When you run a process from the Studio, it runs in standalone mode.
-* Portal mode uses a different container. When you run a process from the Portal, it runs in portal mode.
-This means that if you include a script in the templates, which are the standalone container, the script cannot be called by the process when it runs in portal mode.
-
-To include a script in a process so it can be used in portal mode, there are the following options:
-
-* In the form, add an HTML widget that contains it``.
-
-After a process is xref:admin-application-process-list.adoc[deployed and initialized] in Bonita Portal, it is run by xref:cases.adoc[starting a case].
+process must be configured for the environment in which it runs.
```
Comment (vbotnari12) [inline modules/ROOT/pages/run-a-process-from-bonita-bpm-studio-for-testing.adoc]: If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can select the environment in which to run the process. The process must be configured for the environment in which it runs. =>  A process can only run in the environment(s) in which it has been configured. If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can choose the environment in which to run the process.

File: modules/ROOT/pages/logging.adoc (hunk 1 above)
Comment (baptistemesta) [inline modules/ROOT/pages/logging.adoc]:
```suggestion
Bonita components have their own ways to produce logs, as described below.
```

File: modules/ROOT/pages/logging.adoc (hunk 2 above)
Comment (baptistemesta) [inline modules/ROOT/pages/logging.adoc]:
```suggestion
The Bonita Runtime has two components :
```

File: modules/ROOT/pages/logging.adoc (hunk 3 above)
Comment (baptistemesta) [inline modules/ROOT/pages/logging.adoc]:
```suggestion
In Bonita Studio you have access to the following log files :
```

File: modules/ROOT/pages/multi-language-pages.adoc (hunk 4 above)
Comment (baptistemesta) [reply modules/ROOT/pages/multi-language-pages.adoc]: Next time, you can use the "suggestion" feature to directly propose changes. (and thanks for the english phrasing... we are not great for that 😅  )
Screenshot 2021-09-29 at 10 28 27

Comment (baptistemesta) [inline modules/ROOT/pages/multi-language-pages.adoc]:
```suggestion
By default, translation keys are in en-US english texts. If you want to have custom texts, you can create a new translation section for en-US and handle it as a new translation. This option allows to freely change texts by avoiding duplicates on other language sections.
```

File: modules/ROOT/pages/profile-creation.adoc (hunk 5 above)
Comment (baptistemesta) [inline modules/ROOT/pages/profile-creation.adoc]:
```suggestion
In the mapping section you can map Users / Groups / Roles / Memberships to your profile. You have to map your profile with elements of an active organization. +
```

File: modules/ROOT/pages/profile-creation.adoc (hunk 6 above)
Comment (baptistemesta) [inline modules/ROOT/pages/profile-creation.adoc]:
```suggestion
For adding an application to a profile, open the application and assign the required profile by using xref:applicationCreation.adoc[application descriptor editor].
```

File: modules/ROOT/pages/profiles-applications-overview.adoc (hunk 7 above)
Comment (baptistemesta) [inline modules/ROOT/pages/profiles-applications-overview.adoc]:
```suggestion
By default, it gives access to Bonita Administrator Application, which includes:
```

File: modules/ROOT/pages/run-a-process-from-bonita-bpm-studio-for-testing.adoc (hunk 8 above)
Comment (baptistemesta) [inline modules/ROOT/pages/run-a-process-from-bonita-bpm-studio-for-testing.adoc:17]:
```suggestion
A process can only run in the environment(s) in which it has been configured. If you are using the Enterprise, Performance, Efficiency, or Teamwork editions, you can choose the environment in which to run the process.
```

**Pull request merged**